haskell-for-readers.md

```diff
@@ -759,6 +768,9 @@ From this example, we learn that
  * the function arrow can just as well occur inside an argument, namely when an argument itself is a function.
 
 But what is this type `t`? There is not, actually, a type called `t`. Instead, this is a *type variable*, meaning that the function `twice` can be used with any type. Any lower-case identifier in a type is a type variable (not just `t`), and concrete types are always upper-case.
+{::comment}
+That is not entirely accurate. It's whether they /start/ with a lower- or upper-case letter.
```

Comment:
Right, that’s of course what I try to say. Feel free to propose a better wording :-)